### ISSUE TITLE
Allow users to modify collections from inside the app

### ISSUE THREAD

**Issue by jonbro** [itchio/itch#108]

**Comment by fasterthanlime** [comment]
Since v0.14, you can do that by using the web UI inside the app. Not great, so keeping this issue open, but just in case you didn't know!

(You have to log into the website from the app first)

**Comment by ghost** [comment]
+1 for this, I have a "play later" collection I use to track things I might see when browsing at work or something so I can pull them up quickly on the app at home. Once I've played something and want to remove it, there's no quick way of doing so within the app (and I'm usually too lazy to use the web UI or anything like that, so my list stays cluttered until I happen to feel like editing it in browser, haha)

**Comment by fasterthanlime** [comment]
Still super hyped for this

Still no API support

Still planned!

**Comment by bowlercaptain** [comment]
+1;

I also keep a 'play later' collection, and after installing/playing it would be cool to be able to remove items from the todo list.

Looks like we're maybe waiting on [this issue](https://github.com/itchio/itch.io/issues/291) to be resolved for that to happen?

**Comment by fasterthanlime** [comment]
> Looks like we're maybe waiting on this issue to be resolved for that to happen?

Yup! Although it is in my power to solve that issue as well, I've got a buunch of cleanups to finish up first though.

**Comment by ChildishGiant** [comment]
![image](https://user-images.githubusercontent.com/13716824/84211391-2da6de80-aab3-11ea-9ace-2628152c6bd7.png)
![image](https://user-images.githubusercontent.com/13716824/84211435-444d3580-aab3-11ea-9b36-6d4e6d879e5f.png)

Maybe a tooltip in itch://collections linking to https://itch.io/my-collections to make it clear that it is possible?

**Comment by MarjaE2** [comment]
That's also hard to navigate (due to using one row for Things You Own, which scrolls separately from the rest of the page, which can trigger migraines), and hard to organize (due to requiring users to enter each page to add to collections).
